frontend/pages/hosts/details/cards/HostSummary/HostSummary.tsx

```diff
@@ -174,41 +178,8 @@ const HostSummary = ({
         </div>
         <div className="info-flex__item info-flex__item--title">
           <span className="info-flex__header">Operating system</span>
-          <span className="info-flex__data">{titleData.os_version}</span>
-        </div>
-      </div>
-    );
-  };
-
-  const renderHostDetailsSummary = () => {
-    return (
-      <div className="info-flex">
-        <div className="info-flex__item info-flex__item--title">
-          <span className="info-flex__header">Status</span>
-          <span className={`${statusClassName} info-flex__data`}>
-            {titleData.status}
-          </span>
-        </div>
-        {titleData.issues?.total_issues_count > 0 && renderIssues()}
-        {isPremiumTier && renderHostTeam()}
-        <div className="info-flex__item info-flex__item--title">
-          <span className="info-flex__header">Disk Space</span>
-          {renderDiskSpace()}
-        </div>
-        <div className="info-flex__item info-flex__item--title">
-          <span className="info-flex__header">RAM</span>
-          <span className="info-flex__data">
-            {wrapFleetHelper(humanHostMemory, titleData.memory)}
-          </span>
-        </div>
-        <div className="info-flex__item info-flex__item--title">
-          <span className="info-flex__header">CPU</span>
-          <span className="info-flex__data">{titleData.cpu_type}</span>
-        </div>
-        <div className="info-flex__item info-flex__item--title">
-          <span className="info-flex__header">OS</span>
           <span className="info-flex__data">
-            {isOnlyObserver ? (
+            {isOnlyObserver || deviceUser ? (
```

Comment:
like observer, device user cannot click on operating system to create a policy regarding the operating system